app/controllers/application_controller.rb

```diff
@@ -25,7 +25,12 @@ def append_info_to_payload(payload)
     payload[:host] = request.host
     payload[:request_id] = request.request_id
     payload[:requested_by] = "#{@access_token.owner}-#{@access_token.id}" if @access_token.present?
-    payload[:form_id] = params[:form_id] if params[:form_id].present?
+    payload[:form_id] = if params[:id].present?
```

Comment:
I'm not sure we can use :id like this in the application controller. `id` is the name of the param in the conditions controller and the access_tokens controller.

I guess another approach might be to overide `append_info_to_payload` in the forms controller, calling super and setting the correct form_id value. Not 100% sure that would work.

Reply:
> I'm not sure we can use :id like this in the application controller. id is the name of the param in the conditions controller and the access_tokens controller.

Ah, good catch.

> I guess another approach might be to overide append_info_to_payload in the forms controller, calling super and setting the correct form_id value. Not 100% sure that would work.

I like this idea a lot, I'm going to give it a try.